client/styles/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const mobile = '425px'
+export const tablet = `768px`
+export const largerThan = (size: string): string => `(min-width: ${size})`
```

Comment:
in the long run this should probably be combined with `styles/device.ts`

Reply:
Moved now. Let's keep the codebase as clean as possible ;)